testsuite/config.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
Why is this file required here? The sonobuoy config file is already present in the internal repo where the launcher container files are present.

Reply:
this config.json is required for Data services with Indirect mode validation. This Data services with Indirect mode is not part of launcher container files. So please keep this config.json

Reply:
The [`config.json`](https://github.com/Azure/azure-arc-validation/blob/main/testsuite/config.json) was introduced only for overriding the results directory for sonobuoy versions > 0.55. Why does your config file contain all these settings?

Reply:
Unfortunately I have copied wrong file there . Now I have updated with correct file.